rcl_action/test/rcl_action/test_graph.cpp

```diff
@@ -132,17 +133,14 @@ TEST_F(
   rcl_reset_error();
   // Invalid node name
   ret = rcl_action_get_client_names_and_types_by_node(
-    &this->node, &this->allocator, "_test_this_Isnot_a_valid_name", "", &nat);
-  EXPECT_EQ(RCL_RET_ERROR, ret) << rcl_get_error_string().str;
+    &this->node, &this->allocator, "_!test_this_is_not_a_valid_name", "", &nat);
+  EXPECT_EQ(RCL_RET_NODE_INVALID_NAME, ret) << rcl_get_error_string().str;
   rcl_reset_error();
   // Non-existent node
-  // Note, Opensplice successfully reports graph information about finalized nodes
-  if (!is_opensplice) {
```

Comment:
If we aren't keeping this particular test, then we can remove the definition of `is_opensplice` near the top of the file.